doc/bucket-class-crd.md

```diff
@@ -177,3 +188,46 @@ spec:
       - bs4
       placement: Mirror
 ```
+
+Namespace bucketclass:
+```shell
+noobaa -n noobaa bucketclass create bc namespace --read-resources bs1,bs2 --write-resource bs1
+```
+```yaml
+apiVersion: noobaa.io/v1alpha1
+kind: BucketClass
+metadata:
+  labels:
+    app: noobaa
+  name: bc
+  namespace: noobaa
+spec:
+  namespacePolicy:
+    readResources:
+    - bs1
+    - bs2
+    writeResource: bs1 
+```
+
+Namespace bucketclass:
+```shell
+noobaa -n noobaa bucketclass create bc namespace --read-resources bs1,bs2 --write-resource bs1 --cache-prefix a --cache-ttl 6000
+```
+```yaml
+apiVersion: noobaa.io/v1alpha1
+kind: BucketClass
+metadata:
+  labels:
+    app: noobaa
+  name: bc
+  namespace: noobaa
+spec:
+  namespacePolicy:
+    readResources:
+    - bs1
+    - bs2
+    writeResource: bs1 
+    cache:
+      prefix: 'a'
+      ttl: 6000
```

Comment:
suggestion - design it like we do for backing-stores and namespace-stores:

```
  namespacePolicy:
    type: cache
    cache:
      hub: ns1
      prefix: aaa/
      ttl: 6000
```

```
  namespacePolicy:
    type: merge
    merge:
      writeResource: ns1
      readResources: 
      - ns1
      - ns2
```

```
  namespacePolicy:
    type: s3
    s3:
      resource: ns1
```

Reply:
updated the above suggestion ^

Reply:
thanks for the suggestion, updated.